README.md

```diff
@@ -258,6 +258,31 @@ Similarly to unit tests, for finer control over which tests get executed, e.g. t
 tox -e integration -- tests/integration/test_package_managers.py::test_packages[gomod_without_deps]
 ```
 
+Integration tests verify package managers handlers behavior and ensure that it remains consistent.
```

Comment:
For future reference, README/doc changes should ideally go to separate commits from code changes.

Reply:
Done!